build.gradle

```diff
@@ -212,10 +199,18 @@ allprojects {
     }
     implementation  group: 'com.fasterxml.jackson.module', name: 'jackson-module-jaxb-annotations', version: '2.10.3'
     implementation 'net.minidev:json-smart:2.4.7'
```

Comment:
Most implementation, testImplementation etc. entries use "group:'<group_name>' name:'<name>' version '<version>'" syntax whereas this line (and others in the file) use "'<group_name>:<name>:<version>'".  This is purely cosmetic, but it would look better if a consistent syntax was used throughout the file.

Reply:
addressed

Reply:
There are still a number of instances in the file where `'<group_name>:<name>:<version>'` is used - see lines 209 - 211, 214 - 216 and 238 - 245.

Reply:
addressed these as well

Reply:
Confirmed that format has been changed.